Description: Don't try to detect the version, don't use Pylons theme.
Author: Andrew Shadura <andrewsh@debian.org>

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -19,7 +19,7 @@
 
 import datetime
 import pkg_resources
-import pylons_sphinx_themes
+# import pylons_sphinx_themes
 
 # General configuration
 # ---------------------
@@ -53,7 +53,7 @@
 # other places throughout the built documents.
 #
 # The short X.Y version.
-version = pkg_resources.get_distribution("waitress").version
+version = '1.4.1' # pkg_resources.get_distribution("waitress").version
 # The full version, including alpha/beta/rc tags.
 release = version
 
@@ -100,9 +100,9 @@
 # -----------------------
 
 # Add and use Pylons theme
-html_theme = "pylons"
-html_theme_path = pylons_sphinx_themes.get_html_themes_path()
-html_theme_options = dict(github_url="https://github.com/Pylons/waitress")
+# html_theme = "pylons"
+# html_theme_path = pylons_sphinx_themes.get_html_themes_path()
+# html_theme_options = dict(github_url="https://github.com/Pylons/waitress")
 
 # The style sheet to use for HTML and HTML Help pages. A file of that name
 # must exist either in Sphinx' static/ path, or in one of the custom paths
@@ -172,15 +172,15 @@
 htmlhelp_basename = "waitress"
 
 # Control display of sidebars
-html_sidebars = {
-    "**": [
-        "localtoc.html",
-        "ethicalads.html",
-        "relations.html",
-        "sourcelink.html",
-        "searchbox.html",
-    ]
-}
+# html_sidebars = {
+#     "**": [
+#         "localtoc.html",
+#         "ethicalads.html",
+#         "relations.html",
+#         "sourcelink.html",
+#         "searchbox.html",
+#     ]
+# }
 
 # Options for LaTeX output
 # ------------------------
